src/components/EntityInfo/EntityMenu.vue

```diff
@@ -250,7 +258,16 @@ export default class EntityMenu extends Mixins(AllowableActionsMixin) {
   }
 
   get enableAgmLocationChg (): boolean {
-    return !!GetFeatureFlag('enable-agm-location-chg')
+    return !!GetFeatureFlag('supported-agm-location-chg-entities')?.includes(this.getLegalType)
```

Comment:
I'm wondering if this will fail if the FF is empty (or not returned by LaunchDarkly). Can you test this please (just fake the FF return value).

Reply:
Or...

Add a default value (`[]`) for this FF in feature-flags.ts and remove the conditional chaining on this line.

Reply:
I did test this with empty value. The AGM location change option is not available in the 'more actions'.
![image](https://github.com/bcgov/business-filings-ui/assets/116035339/6eabc209-3937-40e9-8524-327d9737aa07)

Reply:
adding a default value is a better to do. Your thoughts?

Reply:
Good test. Thanks. Looks like the FF check needs to be on the div, since I see where the menu action isn't 😉 

Yes, I think adding the default is better than using conditional chaining.

Reply:
Thanks for the default value.

You can remove the conditional chaining now.

Reply:
And please move the FF check to the list item's parent div, so that you don't get a blank item in the dropdown if the FF is False.

Reply:
both done